.devcontainer/devcontainer.json

```
@@ -0,0 +1,41 @@
+{
+	"name": "Radius Resource Types & Recipes",
+	"image": "mcr.microsoft.com/devcontainers/javascript-node:20-bookworm",
+	
+	"features": {
+		"ghcr.io/devcontainers/features/azure-cli:1": {},
+		// Note : k3d doesn't run too nicely with docker-in-docker for some reason
+        "ghcr.io/devcontainers/features/docker-outside-of-docker:1": {},
+		"ghcr.io/devcontainers/features/terraform:1": {},
+		"ghcr.io/jsburckhardt/devcontainer-features/k3d:1": {},
+		"ghcr.io/devcontainers/features/kubectl-helm-minikube:1": {
+			"minikube": "none",
+			"helm": "none",
+			"kubectl": "latest"
+		},
+		"ghcr.io/dhoeric/features/oras:1": {},
+		"ghcr.io/dapr/cli/dapr-cli:0": {},
+		"ghcr.io/devcontainers-extra/features/apt-packages:1": {
+			"packages": "socat"
+		}
```

Suggestion to add shellcheck since the extension is includeded in the extensions list. Without this I get a warning dialog from the extension saying that it cannot find the extension.
```suggestion
		},
		"ghcr.io/devcontainers-extra/features/shellcheck:1": {}
```